CONTRIBUTING.md

```diff
@@ -1,341 +1,180 @@
 # Contributing
 
-Welcome! As a [Jupyter](https://jupyter.org) project, we follow the [Jupyter contributor guide](https://jupyter.readthedocs.io/en/latest/contributor/content-contributor.html).
+Welcome! As a [Jupyter](https://jupyter.org) project, we follow the [Jupyter
+contributor
+guide](https://jupyter.readthedocs.io/en/latest/contributor/content-contributor.html).
 
-## Setting up minikube for local development
+## Local development for a code contribution
 
-We recommend using [minikube](https://github.com/kubernetes/minikube) for local
-development.
+### Prepare git
 
-1. [Download & install minikube](https://github.com/kubernetes/minikube#installation).
+1. Install [git](https://www.git-scm.com/). To verify it is installed, run this
+   from a terminal.
 
-   For MacOS: You may install minikube using Homebrew `brew cask install minikube` or
-   from a binary at https://github.com/kubernetes/minikube/releases.
-   If you need to install Docker Community Edition (CE) for Mac, please
-   follow the [Docker instructions](https://store.docker.com/editions/community/docker-ce-desktop-mac).
-
-2. [Download & install helm](https://github.com/helm/helm#install).
-
-   You may install Helm using one of the following steps:
-
-   * With the following curl command:
-
-     ```
-     curl https://raw.githubusercontent.com/kubernetes/helm/master/scripts/get | bash
-     ```
-   * From one of the binaries at https://github.com/helm/helm/releases
-   * For MacOS, using Homebrew: `brew install kubernetes-helm`
-
-3. Start minikube.
-
-   For minikube version 0.26 and higher:
    ```bash
-   minikube start
+   git version
    ```
 
-   For older minikube versions:
-   ```bash
-   minikube start --extra-config=apiserver.Authorization.Mode=RBAC
-   ```
+1. Make a GitHub fork of [this
+   repository](https://github.com/jupyterhub/zero-to-jupyterhub-k8s) by creating
+   and then logging into your GitHub account and clicking the Fork button.
 
-   Note on troubleshooting: if you recently upgraded minikube and are now seeing
-   errors, you may need to clear out the `~/.minikube` and `~/.kube` directories
-   and reboot.
-
-4. Use the docker daemon inside minikube for building:
-   ```bash
-   eval $(minikube docker-env)
-   ```
+1. Clone your fork to your local computer.
 
-5. Clone the zero-to-jupyterhub repo:
    ```bash
-   git clone git@github.com:jupyterhub/zero-to-jupyterhub-k8s.git
+   git clone http://github.com/<YOUR-GITHUB-USERNAME>/zero-to-jupyterhub-k8s.git
```

Comment:
Part of why I would rather not have super-detailed step-by-step instructions for how to clone and name remotes is that I disagree with this process. I would always use 

```bash
git clone https://github.com/jupyterhub/zero-to-jupyterhub
cd zero-to-jupyterhub
git remote add mine git@github.com:minrk/zero-to-jupyterhub
```
so that `origin` and most importantly `origin/master` track the true master, and I push to `mine`.

(in reality, I use [hub](https://hub.github.com) and actually do):

```bash
hub clone jupyterhub/zero-to-jupyterhub
hub fork
```

Now, this isn't the only way to do it, but I'm not a big fan of *recommending* people do it the reverse, where `master` doesn't actually track master and `upstream` must be specified for pull. I'd much rather leave cloning steps up to the numerous guides for using git, rather than duplicating them here.